src/plugins/cordova-plugin-device-orientation/sim-host.js

```diff
@@ -32,7 +32,7 @@ module.exports = function (messages) {
         });
         compassWidget.initialize(compass.heading);
 
-        inputHeading.addEventListener('input', function () {
+        inputHeading.addEventListener('change', function () {
```

Comment:
Unrelated telemetry change; the `input` event was causing the value to change more than once while typing a 2- or 3-digit number. With the `change` event, typing `256` in the text entry will not trigger 3 separate events (one for `2`, one for `25` and one for `256`), so I think it's a reasonable improvement